AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/endpoint.model.ts

```diff
@@ -12,10 +12,7 @@ export class EndpointModel {
     displayName?: string;
     sip?: string;
     pin?: string;
-    /**
-     * Defence advocate email address, not their ID
-     */
-    defenceAdvocate?: string;
+    participants_linked?: string[];
```

Comment:
any chance we could change this to `participantsLinked`? Just to stick with the current convention